site/_posts/2017-03-07-deletions-and-dragons.html.md

```diff
@@ -0,0 +1,63 @@
+
+[Janella Silentall][jr], the female Half-Orc Dwarven Defender, sulked. She had been trying to fix Eillyassa's curse for weeks to no avail. Grumbling, she strode off to the edge of the river. She didn't get far without stubbing her toe on a small item that appeared out of nowhere. She yelled ferociously, cursing the deities, and then knelt down to examine the offending item. Forgetting her anger, she read out loud, "Ah! We have [a new rake task][#14107], evm:status_full, that prints the local server and remote servers and their workers ordered by server and zone!"
+
+The company slowly got ready for another day, leaving the coatrack in the middle of the road. [Janella Silentall][jr], the female Half-Orc Dwarven Defender, had attempted idly to courteously move it to the side of the road, but it appeared to be firmly stationary. The flat, open road had begun to have a bit of elevation, and the trees looming forebodingly in the distance promised not shelter but a rather threatening day of watching their collective backs. As the hours of walking crept by, [Jannys Falconsflight][jf], the female Half-Orc Paladin, slowed considerably. She had never dealt well with elevation. The company finally stopped for a short break in the mid-afternoon. [Nerisstina Cupshigh][nc] sat down on a rock with a thud. Frowning, she reached for a shiny looking thing in the dirt at her feet. But the thing didn't move, and, with a heavy sigh, she abandoned it. The dog loped over to see what she'd been digging at, and soon there was a small pit where the shimmer had been. Nerisstina smoothed the dirt over, and, with a gasp, pulled out a small object to examine it. "Ah!", she exclaimed. "Curiouser and curiouser. It's apparently a [backlog method to PglogicalSubscription objects][#14010]!" 
+
```

Comment:
This is my favorite part.  @carbonin:  shiny objects and excitement for backlog methods!